src/index.html

```diff
@@ -117,9 +117,6 @@ <h3 class="text--headline text--headline--3 space--mt--large space--mb">The Do
           <li><strong>Do</strong> the outer work and figure out how to change the oppressive systems</li>
           <li><strong>Do</strong> amplify (online and when physically present) the voices of those without your privilege</li>
           <li><strong>Do</strong> learn how to <em>listen</em> and accept criticism with grace, even if it’s uncomfortable</li>
-          <li><strong>Do</strong> realize you are the collective presence of people who cannot safely stand alone</li>
```

Comment:
Hi @jagawag! Thanks for this removing these edits. I made edits to the language after your first PR, but would love to have a convo about your thinking behind adding these changes. Let me know!